src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -220,16 +232,7 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
             SelectVersion();
 
             OnPropertyChanged(nameof(Versions));
-            OnPropertyChanged(nameof(VersionsView)); // This can change the selected version if a version is added after SelectedVersion is set
-
-            // If the selected version changed, reset it to the correct one
-            if (_nugetProjects.Any() &&
-                IsProjectPackageReference &&
-                SelectedVersion != FirstDisplayedVersion)
-            {
-                SelectedVersion = FirstDisplayedVersion;
-                UserInput = SelectedVersion?.ToString();
-            }
+            OnPropertyChanged(nameof(VersionsView));
```

Comment:
Do we really need this? Why was the comment removed - does it no longer apply?

Reply:
Yes, we need this, when we add the items and filtering, we need to do a refresh, otherwise the combobox list will be empty